File: Bulgarian Competitions/BNOI - Group D/Mini Poker/Mini Poker.cs
using System;
using System.Linq;

class MiniPoker
{
    static void Main()
    {
        dynamic[] input = Console.ReadLine().Split(' ');
        int[] numbers = new int[5];
        for (int i = 0; i < input.Length; i++)
        {
            numbers[i] = int.Parse(input[i]);
        }

        Array.Sort(numbers);
        bool isSeq = true;
        for (int i = 1; i < numbers.Length; i++)
        {
            if (numbers[i] != numbers[i - 1] + 1) isSeq = false;
        }

        Console.WriteLine((isSeq) ? String.Join(" ", numbers) : numbers.Max().ToString());
    }
}

File: Bulgarian Competitions/BNOI - Group D/Mini Poker/Mini Poker.js
function solve(args) {
    var numbers = args[0].split(' ');
    for (var i = 0; i < numbers.length; i++) {
        numbers[i] = parseInt(numbers[i]);
    }

    numbers.sort(function (a, b) { return a - b });
    var isSeq = true;
    for (var i = 1; i < numbers.length; i++) {
        if (numbers[i] != numbers[i - 1] + 1) isSeq = false;
    }

    return (isSeq) ? numbers.join(' ') : Math.max.apply(null, numbers);
}